services/bettersocial/user/BetterSocialBlockUserV2.js

```diff
@@ -65,7 +65,7 @@ const BetterSocialBlockUserV2 = async (token, selfUserId, targetUserId, source,
           selfUserId,
           targetAnonymousUserId?.user_id,
           source,
-          {transaction: t, message, postId, reason, isAnonymous: true}
+          {transaction: t, message, postId: '', reason, isAnonymous: true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `postId` argument in the `BetterSocialScoreBlockUser` function call has been replaced with an empty string. This could potentially lead to incorrect behavior if the function relies on a valid `postId`. Please ensure that this change is intentional and won't cause any issues.
```diff
- {transaction: t, message, postId, reason, isAnonymous: true}
+ {transaction: t, message, postId: '', reason, isAnonymous: true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
mas @n-permana why is this postId set to empty string?